configure.ac

```diff
@@ -40,7 +40,7 @@ MONO_VERSION_BUILD=`echo $VERSION | cut -d . -f 3`
 # This can be reset to 0 when Mono's version number is bumped
 # since it's part of the corlib version (the prefix '1' in the full
 # version number is to ensure the number isn't treated as octal in C)
-MONO_CORLIB_COUNTER=17
+MONO_CORLIB_COUNTER=18
```

Comment:
There is contention over this number currently.
At least three of us are iterating here.
@kyte is in the high teens.
I am in the low 20s.
I suggest you move to 30.